debian/.gitignore

```diff
@@ -0,0 +1,2 @@
+changelog
```

Comment:
why so?

Reply:
It is generated during the build from changelog.ex template (the next comment).
Unfortunately, Debian package cannot be built without changelog.
This changelog has to have special structure, so original CHANGELOG.md can't be used, and is intended to represent changes to package itself, and not to the project.
So we either generate one from scratch each time the package is built or keep adding new records to it with similar comments like "we have not changed anything, just building the package once again - _signature_, date"

Reply:
I'd rather have some proper `changelog` that can be used right away, `Makefile` can edit it for auto-built versions, but otherwise it'd be maintained manually (yeah, with "New upstream release" messages, maybe with some debian-specific ones if/when needed).